loader/loader.go

```diff
@@ -111,17 +111,22 @@ func (l *Loader) walkDirectoryCallback(path string, info os.FileInfo, err error)
 
 		key = strings.Replace(key, "/", ".", -1)
 		stringValue := string(contents)
-		entry := entry.New(stringValue, 0, false)
+		e := &entry.Entry{
```

Comment:
other than eliminating the function call, why eliminate New in favor of this?

Reply:
Function call is not that important, this is probably inlined by the compiler. There were two things removing `New` avoids. 

First, with the addition of `Modified`, the signature would have to be changed to include a fourth parameter. This would require the same quantity of code changes to update everything to support the new parameter, but would ultimately lead to a more unwieldy API for creating an `Entry` (which should really only be created by this library). 

Second, `New` had no extra logic - it just created the struct - when callers can instead leverage the struct literal and its zero values to initialize instead. Instances of `Entry` can also be created without the constructor anyways, so `New` doesn't protect from creating an invalid `Entry` (which is not possible).

Hypothetically, where we are calling:

```go
e := entry.New("foo", 0, false)
```

One could instead do, with the same effect:

```go
e := &entry.New{StringValue:"foo"}
```

Reply:
thanks for the explanation!